.github/workflows/publish_docs.yml

```diff
@@ -1,47 +1,67 @@
+# WARNING: DO NOT EDIT!
+#
+# This file was generated by plugin_template, and is managed by it. Please use
+# './plugin-template --github pulp_npm' to update this file.
+#
+# For more info visit https://github.com/pulp/plugin_template
+---
 name: Publish docs
 
 on:
+
   push:
     branches:
       - master
     tags:
```

Comment:
The docs are published during the release workflow so this doesn't really seem necessary? 

And publishing them on every push seems like overkill when we already are publishing them nightly

Reply:
> The docs are published during the release workflow so this doesn't really seem necessary?

Agreed, I will remove that.

> And publishing them on every push seems like overkill when we already are publishing them nightly. Or at least we don't need to publish them nightly if we already do so every time the branch is updated.

So this is the current behavior in Travis. I'm not sure why we do this.